### PR TITLE
Scavengers: Update /scavhelp

### PR DESCRIPTION
**Pull request by aQrator**
the /scavhelp command has been outdated for a long time. This change intends to update the command to better represent how the room functions.

### CODE REVIEW

File: server/chat-plugins/scavengers.ts [hunk 1]
```diff
@@ -2541,61 +2541,76 @@ export const commands: ChatCommands = {
...
+			"- /joinhunt: Join the current scavenger hunt.",
+			"- /leavehunt: Leave the current scavenger hunt. Also resets your progress.",
+			"- /viewhunt: Show the ongoing hunt up to where you solved it.",
+			"- /scavenge <em>[guess]</em>: Submit your answer to the current hint.",
+			"- /scavengerstatus  (or /scav status): Check your status in the current hunt.",
+			"- /scavengers queue (or /scav queue): Showcase the hunts currently in queue, with the answers hidden for any hunt that is not yours.",
+			"- /scavengerhint (or /scav hint): View your latest hint in the current game.",
+			"- /scavladder (or /scav top): View the current bimonthly scavengers leaderboard.",
+			"- /scavrank <em>[user]</em>: View the rank of the user on the monthly scavenger leaderboard. Defaults to the user if no name is provided.",
+			"For a more in-depth overview, use /roomhelp staff."
```
Comment (mia-pi-git) [inline server/chat-plugins/scavengers.ts:2554]:
```suggestion
			"For a more in-depth overview, use /roomhelp staff.",
```

File: server/chat-plugins/scavengers.ts [hunk 2]
```diff
@@ -2541,61 +2541,76 @@ export const commands: ChatCommands = {
...
-			"- /scav twists - shows a list of all the twists that are available on the server.",
-			"- /scav settwist <em>[twist name]</em> - sets the default twist mode for all official hunts. (Requires: # &)",
-			"- /scav resettwist - resets the default twist mode for all official hunts to nothing. (Requires: # &)",
-			"- /starttwist(hunt/practice/official/mini/unrated) <em>[twist] | [host] | [hint] | [answer] | [hint] | [answer] | [hint] | [answer] | ...</em>  - creates a new regular scavenger hunt that uses a twist mode in the specified game type.  This can be used inside a scavenger game mode.",
-			"- /nexthunt - starts the next hunt in the queue.",
-			"- /recycledhunts - Modify the database of recycled hunts and enable/disable autoqueing them. More detailed help can be found in /recycledhuntshelp",
+			"<strong>Staff and auth commands:</strong>",
+			"As a <strong>room voice (+)</strong>, you can use the following Scavengers commands, on top of the regular commands (see /scavhelp):",
+			"- /scav edithunt <em>[question number]<\em>, <em>[hint | answer]</em>, <em>[value]</em>: Edit the ongoing scavenger hunt. Only the host(s) can edit the hunt.",
```
Comment (mia-pi-git) [inline server/chat-plugins/scavengers.ts:2559]:
```suggestion
			"- /scav edithunt <em>[question number]</em>, <em>[hint | answer]</em>, <em>[value]</em>: Edit the ongoing scavenger hunt. Only the host(s) can edit the hunt.",
```

File: server/chat-plugins/scavengers.ts [hunk 3]
```diff
@@ -2541,61 +2541,76 @@ export const commands: ChatCommands = {
...
+			"- /scav viewqueue (or /scav queue): Look at the list of queued scavenger hunts. Now also includes the option to remove hunts from the queue.",
+			"- /resethunt: Reset the current scavenger hunt without revealing the hints and answers, nor giving out points.",
+			"- /resethunttoqueue: Reset the ongoing scavenger hunt without revealing the hints and answers, nor giving out points. Then, add it directly to the queue.",
+			"- /scav timer <em>[minutes]</em>: Set a timer to automatically end the current hunt. Setting [minutes] to 0 turns off the timer.",
+			"- /endhunt: End the current scavenger hunt immediately and announce the winners and the answers.",
+			"- /nexthunt: Start the next hunt in the queue.",
+			"- /viewhunt: View the ongoing scavenger hunt. As a host, you can also view the hunt in its entirety.",
+			"- /inherithunt: Become the staff host, gaining staff permissions to the current hunt.",
+			"- /scav games create <em>[game mode]</em>: start a game of the given mode.",
```
Comment (mia-pi-git) [inline server/chat-plugins/scavengers.ts:2575]:
```suggestion
			"- /scav games create <em>[game mode]</em>: start a game of the given mode.",
```

File: server/chat-plugins/scavengers.ts [hunk 4]
```diff
@@ -2541,61 +2541,76 @@ export const commands: ChatCommands = {
...
+			"- /scav queuerecycled <em>[number]</em>: Queue a recycled hunt from the database. If <em>[number]</em> is left blank, then a random hunt is queued.",
+			"- /recycledhuntshelp: give more info about the recycled hunts.",
+			"<br />As a <strong>room owner (#)</strong>, you can also use the following scavengers commands:",
+			"- /scav resetladder: Reset the monthly scavenger leaderboard.",
+			"- /scav setpoints <em>[1st place]</em>, <em>[2nd place]</em>, <em>[3rd place]</em>, <em>[4th place]</em>, <em>[5th place]</em>, ...: Set the point values for wins of officials, minis and regular hunts.",
+			"- /scav defaulttimer <em>[value]</em>: Set the default timer applied to automatically started hunts from the queue.",
+			"- /scav setblitz <em>[value]</em> ...: Set the blitz award to the given value.",
+			"- /scav settwist <em>[twist name]</em>: Set the default twist mode for all official hunts.",
+			"- /scav resettwist: Reset the default twist mode for all official hunts to nothing.",
+			"- /scav modsettings: Allow or disallow miscellaneous room settings"
```
Comment (mia-pi-git) [inline server/chat-plugins/scavengers.ts:2596]:
```suggestion
			"- /scav modsettings: Allow or disallow miscellaneous room settings",
```

**Comment by Zarel** [comment]
It seems bad to change these from `-` to `:`, when every other help command in the entire sim still uses `-`.

We can maybe discuss changing them everywhere, but just in one command seems wrong to me.

**Comment by Zarel** [comment]
I guess help commands are currently really inconsistent, anyway. We'll just let this be someone else's problem.

**Comment by aQrator** [comment]
I mostly copied the style that the /roomhelp staff employs. I believe that, with the exception of one that I think is redundant, all the scavengers help commands should be consistent now. I'll take a day soon(tm) to sweep through the other scavengers explanation lines and make sure they are consistent too.

**Comment by Zarel** [comment]
I guess I shouldn't be _surprised_ that other commands are inconsistent, too. :/

It'll all need to be fixed up at some point! Until then, feel free to do as you like.

**Pull request merged**